### PR TITLE
dockerfile: add comments

### PR DESCRIPTION
**Pull request by jzelinskie**

### CODE REVIEW

**Comment by mrd0ll4r** [comment]
This apparently needs a rebase? No idea why, but apart from that LGTM.

**Comment by jzelinskie** [comment]
Yea, dunno why. Apparently there's an "update branch" button on GitHub and I clicked it to see what it did. Merged master into this branch. I'm going to rebase later today to fix it.

**Comment by jzelinskie** [comment]
rebased, @mrd0ll4r ptal

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+# Install OS-level dependencies
+RUN apk update && \
+    apk add curl git && \
+    curl https://glide.sh/get | sh
 
+# Copy our source code into the container
 WORKDIR /go/src/github.com/chihaya/chihaya
 ADD . /go/src/github.com/chihaya/chihaya
+
+# Install our golang dependencies and compile our binary.
```
Comment (mrd0ll4r) [inline Dockerfile]: This is the only comment ending in a period. We must have them all end the same way.

**Comment by jzelinskie** [comment]
ptal

**Pull request merged**